### ISSUE TITLE
[a11y] Links must be underlined

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#36117]
### Steps to reproduce the issue
Set the colour to monochrome for atum.

Find the link here: 
![image](https://user-images.githubusercontent.com/1035262/143474275-10568f6a-e9af-4790-8179-e4883205f0b1.png)

### Expected result
The user can see where is a link (underlined)

**Comment by Gaurav05082002** [comment]
can you elaborate a little more , which links are to be underlined ,

**Comment by chmst** [comment]
You see .. that you cannot see the link - because I used monochrome display

It is the last line "update your site by manually..." Maybe there are more such links in other "empty state" sceen and a generic approach is needed?



<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36117">issues.joomla.org/tracker/joomla-cms/36117</a>.</sub>

**Comment by richard67** [comment]
Closing as having a pull request. Please test #36134 . Thanks in advance.
